tests/e2e/specs/admin/4-refund-label.test.js

```diff
@@ -53,10 +57,10 @@ describe( 'Refund shipping label', () => {
 	it( "doesn't work for expired labels", async () => {
 
 		await withExpiredShippingLabelAndOrder( async ( { order } ) => {
-			console.log( `# Opening order page for order #${order.id}` );
+			// console.log( `# Opening order page for order #${order.id}` );
```

Comment:
Should we just delete these commented out lines (if they're really not needed)?

Reply:
I am definitely not a fan of leaving in commented out code; however, these `console.log`s are pretty invaluable when it comes to debugging these tests and I left that in because doing this might not be immediately obvious to the next person who has to look at this and modify these tests (or at least it wasn't to me). Despite that defence, I will gladly remove them if you think so, because they are currently inactive.

Reply:
Actually, I changed my mind. Just removed these goofy comments. Good riddance!

Reply:
Haha. Sounds good! If there was something really special worth keeping maybe a comment explaining why it was left there would be good.